CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changes
 
+## 0.19
```

Comment:
I propose omitting this line or changing it to `0.19.2` or `NEXT_RELEASE`

Reply:
Done